Makefile

```diff
@@ -108,4 +110,20 @@ check-fmt:
 
 .PHONY: e2e-test
 e2e-test:
-	cd tests/e2e && make e2e-test
\ No newline at end of file
+	cd tests/e2e && make e2e-test
+
+# Command for in-place substitution
+SED_CMD := sed -i$(shell uname -s | grep -q Darwin && echo " ''")
+
+# Create list of paths to Go, YAML, README.md, and Makefile files
+FILE_PATHS := $(shell find . -type f \( -name "*.go" -o -name "*.yaml" -o -name "README.md" -o -name "Makefile" \))
```

Comment:
With this change you have the ability to override version information in go file at the build time which didn't exist earlier so it made sense to pass it when running the build command. Now passing this information is redundant now and only serves the process of storing this in build metadata.

Not only that you can simply run go build (first instinct for any Go engineer), you will have version information instead of "Filled by the build system" .

Reply:
As discussed offline, addressed review comment.